content/doc/book/managing/plugins.adoc

```diff
@@ -85,6 +85,9 @@ configured Update Center.
 [[install-with-cli]]
 === Using the Jenkins CLI
```

Comment:
```suggestion
=== Using the {JOSS} CLI
```
```suggestion
=== Using the Jenkins CLI
```

Reply:
Thanks for taking the time to comment @cari-mason !  Much appreciated.

In this case, the review is much broader scoped than the proposed change.  We prefer to keep reviews focused on the proposed change rather than broadening them to include surrounding content.

If surrounding content needs to be changed, the preferred technique is to submit another pull request that updates the other content.  That change can then be reviewed and discussed separately.

Concerning your specific suggestion, we don't use acronyms to refer to Jenkins.  Using "(JOSS)" to refer to Jenkins is not a valid reference.  Using "Jenkins OSS" is not a valid reference.  All of Jenkins is open source.  Appending "OSS" to the end of the "Jenkins" name makes it seem that there might be some Jenkins that is not open source.  That could mislead users, since Jenkins is all open source.

Reply:
jenkins does not have this macro unfortunately